libs/ColorKit/include/internal/conversion.h

```diff
@@ -0,0 +1,23 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// Based on work by @gurki - https://github.com/gurki/vivid
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "RGB.h"
+#include "types.h"
+
+namespace leka::ColorKit::internal {
+
+auto clamp(const float &color) -> uint8_t;
+
+auto fromOklab(const oklab_t &) -> lrgb_t;
+
+auto fromLrgb(const lrgb_t &) -> oklab_t;
+
+auto rgb2oklab(const RGB &color) -> oklab_t;
+
+auto oklab2rgb(const oklab_t &color) -> RGB;
```

Comment:
tu as pas mis les noms des args au dessus, enlève les ici ou mets les partout

```suggestion
auto rgb2oklab(const RGB &) -> oklab_t;

auto oklab2rgb(const oklab_t &) -> RGB;
```